developer-docs-site/docs/nodes/run-a-local-testnet.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Validator on Local Testnet"
+title: "Run a Local Testnet"
 slug: "run-a-local-testnet"
 sidebar_position: 9
 ---
 import BlockQuote from "@site/src/components/BlockQuote";
 
-# Validator on Local Testnet
+# Run a Local Testnet
```

Comment:
Change both this and the title key in the above front matter to: **Run a Local Testnet with Validator**
(w in with is lowercase)